CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+[asdf-vm]: https://asdf-vm.com/
+
+# 🚀 Getting Started
+
+These instructions will get you a copy of the project up and running on your
+local machine for development and testing purposes.
+
+## 📥 Prerequisites
+
+The following software is required to be installed on your system:
+
+- [Node.js 16.15+](https://nodejs.org/en/download/)
+
+We recommend using [asdf version manager][asdf-vm] to install and manage all the programming languages' requirements.
+
+## 🗂️ Resources
+
+We use the following open-source software:
+
+- [Next.js](https://nextjs.org/) (using [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app))
+
+- [React-Big-Calendar](https://github.com/jquense/react-big-calendar)
+
+- [Ant Desgin](https://ant.design/docs/react/introduce) ([Documentation](https://github.com/ant-design/ant-design/))
+
+- [EmailJs](https://www.emailjs.com/)
+
+- [html2pdf](https://github.com/eKoopmans/html2pdf.js)
+
+- [jsPDF](https://github.com/parallax/jsPDF)
```

Comment:
I don't think this libraries should be here. This should be a place for major libraries, otherwise this list will increase abnormally 😅